cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -110,6 +110,7 @@
     public static final String ANNO_STRIMZI_IO_STORAGE = Annotations.STRIMZI_DOMAIN + "storage";
     public static final String ANNO_STRIMZI_IO_DELETE_CLAIM = Annotations.STRIMZI_DOMAIN + "delete-claim";
     public static final String ANNO_STRIMZI_LOGGING_HASH = Annotations.STRIMZI_DOMAIN + "logging-hash";
+    public static final String ANNO_STRIMZI_LOGGING_APPENDERS_HASH = Annotations.STRIMZI_DOMAIN + "logging-appenders-hash";
```

Comment:
Do we need this? It seems the annotation is added, but never used? If we do need it then I think a little Javadoc about how it's used wouldn't go amiss.

Reply:
It is used in KAO

Reply:
Yeah, it's added there, (https://github.com/strimzi/strimzi-kafka-operator/pull/3469/files#diff-f19c7c3bdf294affc86939228666be84R2458), but never read. Why is it needed if it's never read? Is it simply so that we can tell that the logging was changed? If so then add a comment explaining how that works.

Reply:
It changes the annotation an thus kafka pods are rolled (appenders are not changeable dynamically). I will add a doc.

Reply:
Thanks, I know it makes sense to us, here and now, but I for one will forget this within about 5 minutes and stuff like this makes it harder for people who are not familiar with it to understand what the purpose of this is. Can you also comment the other hash which is used in the same way?